client/packages/common/src/ui/components/inputs/DatePickers/DatePickerInput/DatePickerInput.tsx

```diff
@@ -33,6 +37,8 @@ export const DatePickerInput: FC<DatePickerInputProps> = ({
       width={width}
       error={error}
       label={label}
+      minDate={minDate}
```

Comment:
The date part works fine - but I'm able to get the time to be more than it should be

<img width="481" alt="image" src="https://github.com/openmsupply/open-msupply/assets/9192912/8a2ed0f3-be08-46a6-9328-8011aa17600e">

Reply:
Ah, good catch. It looks like the Mui component is not disabling the illegal values for the time picker like it does for the Date component.

Rather than debugging the Material component, I've just put an additional check on the "onChange" function -- if the time is outside the mix/min, it just changes it to the max/min.

d31c498f6